xhr.bs

```diff
@@ -1659,8 +1638,8 @@ method steps are:
 <ol>
  <li><p>Let <var>value</var> be <var>value</var> if given; otherwise <var>blobValue</var>.
 
- <li><p>Let <var>entry</var> be the result of <a lt="create an entry">creating an entry</a> with
- <var>name</var>, <var>value</var>, and <var>filename</var> if given.
+ <li><p>Let <var>entry</var> be the result of <a for="entry list" lt="create an entry">creating an
```

Comment:
No newline inside `<a>`. You might also be able to drop the `lt` attribute here. I think Bikeshed improved since this was written.